src/commands/addBinding/BindingCreateStep.ts

```diff
@@ -27,7 +27,9 @@ export class BindingCreateStep extends AzureWizardExecuteStep<IBindingWizardCont
         binding.direction = bindingTemplate.direction;
 
         for (const b of bindingTemplate.settings) {
-            binding[b.name] = getBindingSetting(context, b);
+            if (getBindingSetting(context, b) !== "undefined") {
```

Comment:
Is `"undefined"` (a string) the actual value or are you looking for `undefined` (the symbol)? Is `null` relevant as well?

Reply:
I am pretty sure it should be a string in the case of `partitionKey` and `collectionThroughput` settings since we are getting the value from user input and from the binding settings it also seems to be a string. Another issue I noticed is that the current behavior of the input box has undefined as the placeholder for `partitionKey` and `collectionThroughput` which may also want to be changed since its an invalid input. I tried inputting null and it also caused issues so I will add that, but not sure if undefined the symbol is used anywhere.